### PR TITLE
Run `task testjs` on CI

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Run changes in the `test.js` file on CI for both MongoDB and FerretDB.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [ ] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by rumyantseva — DISMISSED** [review]
`:ack:`

File: .github/workflows/js.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+name: JS
+on:
+  pull_request:
+    types:
+      - unlabeled # if GitHub Actions stuck, add and remove "no ci" label to force rebuild
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - build/legacy-mongo-shell/*.js
```
Comment (b1ron) [inline .github/workflows/js.yml:11]: This will run the example file too?

Comment (b1ron) [reply .github/workflows/js.yml:11]: Oh it's to make the workflow run.

**Review by b1ron — APPROVED** [review]
Looks good!

**Review by w84thesun — APPROVED** [review]

**Pull request merged**